test/unit/helpers/snapshot.test.ts

```diff
@@ -50,13 +50,13 @@ describe('snapshot', () => {
     });
 
     it.each(expectedShortBody)('truncates the body when necessary', async data => {
-      const results = await testSnapshot.getProposals('', data.length);
+      const results = await testSnapshot.getProposals([''], data.length);
```

Comment:
Why this is empty?

Reply:
We don't test the params, the returned value from the hub is mocked.

Tests are only testing the formatting of the returned value

Reply:
Can it be `[]` then?

Reply:
typescript is expecting an array of strings

Reply:
Hum weird, you see an error if you use empty array?